airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java

```diff
@@ -40,13 +50,16 @@ public class OAuthHandler {
   private final ConfigRepository configRepository;
   private final OAuthImplementationFactory oAuthImplementationFactory;
   private final TrackingClient trackingClient;
+  private final SecretsRepositoryReader secretsRepositoryReader;
 
   public OAuthHandler(final ConfigRepository configRepository,
                       final HttpClient httpClient,
-                      final TrackingClient trackingClient) {
+                      final TrackingClient trackingClient,
+                      final SecretsRepositoryReader secretsRepositoryReader) {
     this.configRepository = configRepository;
     this.oAuthImplementationFactory = new OAuthImplementationFactory(configRepository, httpClient);
     this.trackingClient = trackingClient;
+    this.secretsRepositoryReader = secretsRepositoryReader;
   }
 
   public OAuthConsentRead getSourceOAuthConsent(final SourceOauthConsentRequest sourceDefinitionIdRequestBody)
```

Comment:
This wasn't introduced by this PR, but this parameter should probably be named something like `sourceOauthConsentRequestBody` instead of `sourceDefinitionIdRequestBody`. Same for the destination method